content/kubermatic/master/installation/install_kkp_CE/add_seed_cluster_CE/_index.en.md

```diff
@@ -189,19 +218,19 @@ Output will be similar to this:
 #seed-proxy-kubermatic-6dd5cc95cf-r6wvb                 1/1     Running   0          80m
 ```
 
+If you experience issues with the seed cluster setup, for example nothing happening in the `kubermatic` namespace,
+check the Kubermatic Operator's logs, for example via `kubectl --namespace kubermatic logs -f kubermatic-operator-7f6957869d-89g55`.
```

Comment:
Once again, it would be useful to mention that this should be run on the master cluster.